computeGlobalConservation.py

```diff
@@ -28,5 +28,6 @@ def computeGlobalConservation(varConS, varConT, varSS, varS2T, varST, areaS, are
        
        # Compute the global conservation metric
        L_g = (L_S2T - L_SS) / L_ST
+       # L_g = (L_S2T - L_ST) / L_SS
```

Comment:
Is this the modification you were talking about looking at?

Reply:
Yes. I basically compares the projection from source-to-target against the target integral. But it is flawed because we expect the remap scheme to conserve the source data only and it will be consistent with this definition only if source and target integrals are exactly the same.

Reply:
Wouldn't you want the denominator to be L_SS as well then? Or actually wouldn't we want to have: (L_S2T - L_ST) / L_ST = L_S2T / LST - 1 ? This way every term is integrated on the same mesh with the same error, which would cancel leaving only the relative change from the remap.

Reply:
I agree that you would not want what is commented out here. This would compare L_S2T which was calculated with L_SS as the target total integral.
No reason to believe L_ST describes that same quantity.

Reply:
I guess I'm wondering if the commented out portion is useful to have in the code or will just lead to more confusion.

Reply:
@jeguerra The denominator is just the scalar scaling term. We care more about what is being compared in the numerator. My thought process there was that the L_SS is the reference and L_S2T gives the departure away from L_ST (on target grid), if LST=L_SS.

Reply:
@kuberry Yeah I can remove it. I put it in there to precisely start this discussion so that everyone is on the same page :)

Reply:
@vijaysm If the denominator doesn't matter then lets set it to 1 and save a computation. Although, I think in the uncommented line L_SS should be the normalizing scalar as that's the number we're measuring away from.

Reply:
@jeguerra The denominator does matter if you actually are not globally conservative. We can still verify consistency i.e., if the global integral actually converges to exact value at the remap order as grid is refined.